LiskHQ/lisk-desktop
# Fix state update warning on wallet page - Closes #4029

### What was the problem?

#4029

### How was it solved?

It was solved by moving the `showWarning` function call from within the return statement in overview/index.js to a useEffect

### How was it tested?

- Login on the lisk desktop application using the web
- Visit the wallet page
- On the browser's console, It would be observed that there is no `Cannot update during an existing state transition (such as within 'render'). Render methods should be a pure function of props and state.` warning thrown.
